content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -13,29 +13,43 @@ You need to install a
 into each node in the cluster so that Pods can run there. This page outlines
 what is involved and describes related tasks for setting up nodes.
 
-<!-- body -->
 
 Kubernetes {{< skew currentVersion >}} requires that you use a runtime that
-conforms with the 
+conforms with the
 {{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI).
 
 See [CRI version support](#cri-versions) for more information.
 
-This page lists details for using several common container runtimes with
-Kubernetes, on Linux:
+This page provides an outline of how to use several common container runtimes with
+Kubernetes.
 
 - [containerd](#containerd)
 - [CRI-O](#cri-o)
 - [Docker Engine](#docker)
 - [Mirantis Container Runtime](#mcr)
 
 {{< note >}}
-For other operating systems, look for documentation specific to your platform.
```

Comment:
The original content here at least mentions other operating systems. Where this information is relocated?

Reply:
Do you mean OSs other than Linux or Windows?

My thinking is that, given our docs policy on 3rd party content, the container runtimes' docs should cover installs onto various operating systems. If they don't, folks can propose adding it to _those_ docs, not to Kubernetes.